src/main/resources/api.yaml

```diff
@@ -87,7 +87,117 @@ paths:
           description: User is not authorized
         404:
           description: Project not found
-  /projects/{id}/sprints:
+  /projects/{projectId}/members:
+    get:
+      tags:
+        - projects
+      summary: Retrieves users belonging to the project
+      operationId: getUsersFromProject
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+      responses:
+        200:
+          description: Users retrieved successfully
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/ProjectMemberDto'
+        404:
+          description: Project not found
+    post:
+      tags:
+        - projects
+      summary: Adds a user to the project
+      operationId: addUserToProject
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+        - in: body
+          name: request
+          description: Information identifying the user
+          required: true
+          schema:
+            $ref: '#/definitions/UserIdentificationDto'
+      responses:
+        200:
+          description: User added successfully
+          schema:
+            $ref: '#/definitions/ProjectMemberDto'
+  /projects/{projectId}/members/{userId}:
```

Comment:
Ja się tu jeszcze ogarniam, sorki. To `userId` to faktycznie jest `userId`, a nie `userInProjectId`?

Reply:
Mam bardzo dużą nadzieję, że istotnie tak jest :worried: